.github/workflows/mypy_primer_comment.yml

```diff
@@ -27,18 +27,20 @@ jobs:
                repo: context.repo.repo,
                run_id: ${{ github.event.workflow_run.id }},
             });
-            const [matchArtifact] = artifacts.data.artifacts.filter((artifact) =>
-              artifact.name == "mypy_primer_diffs");
+            const matchArtifacts = artifacts.data.artifacts.filter((artifact) =>
+              artifact.name.startsWith("mypy_primer_diff"));
 
-            const download = await github.rest.actions.downloadArtifact({
-               owner: context.repo.owner,
-               repo: context.repo.repo,
-               artifact_id: matchArtifact.id,
-               archive_format: "zip",
-            });
-            fs.writeFileSync("diff.zip", Buffer.from(download.data));
+            for (const matchArtifact of matchArtifacts) {
+              const download = await github.rest.actions.downloadArtifact({
+                 owner: context.repo.owner,
+                 repo: context.repo.repo,
+                 artifact_id: matchArtifact.id,
+                 archive_format: "zip",
+              });
+              fs.writeFileSync(`${matchArtifact.name}.zip`, Buffer.from(download.data));
+            }
 
-      - run: unzip diff.zip
+      - run: for file in *.zip; do unzip $file; done
```

Comment:
Does this not work?
```suggestion
      - run: unzip *.zip
```

Reply:
No. `unzip` expects the first argument to be a zip file and the remaining arguments to be archive members to unzip.